R/get_fs_int.R

```diff
@@ -0,0 +1,62 @@
+#' Compute interaction indicators for tspa() function
+#' 
+#' @param dat A data frame containing first-order factor score indiactors.
+#' @param fs_name A character vector indicating names of factor scores variables.
+#' @param se A vector indicating standard error of factor scores
+#' @param loading A vector indicating model-implied loadings of factor scores
+#' @param model An optional string specifying the measurement model
+#'              in \code{lavaan} syntax.
+#' @return A data frame of product indicators for interaction terms,
+#'         with their loadings and standard errors.
+#'
+#' @importFrom utils combn
+#'
+#' @export
+
+get_fs_int <- function(dat, fs_name, se, loading, model = NULL) {
```

Comment:
To be consistent with the arguments in `tspa()`

```suggestion
get_fs_int <- function(dat, fs_name, se_fs, loading_fs, model = NULL) {
```

Reply:
For now, we can force users to have `se_fs` and `loading_fs` to be columns in `dat`, and make sure that the inputs are character vectors.

Reply:
Got it. Thanks!